core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -307,7 +308,7 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
     |  driverExtraLibraryPath  $driverExtraLibraryPath
     |  driverExtraJavaOptions  $driverExtraJavaOptions
     |  supervise               $supervise
-    |  queue                   $queue
+    |  queue                   $queue(YARN Only)
```

Comment:
the `(YARN Only)` format seems a bit unusual here? is there any other YARN only argument and how are they documented?